packages/core/src/lib/select/select.spec.svelte

```diff
@@ -0,0 +1,30 @@
+<!-- 
+  @component
+
+  This component allows us to render a Tooltip with its slotted
+  children, due to limitations with rendering slots using
+  `@testing-library`.
+
+  See:
+  - https://sveltesociety.dev/recipes/testing-and-debugging/unit-testing-svelte-component
+  - https://github.com/testing-library/svelte-testing-library/issues/48
+ -->
+<script lang="ts">
+import type { InputState } from '$lib/input/input.svelte';
+import Select from './select.svelte';
+
+export let value: string | undefined = undefined;
+export let disabled = false;
+export let state: InputState = 'none';
```

Comment:
```suggestion
export let state: InputState | undefined = 'none';
```
Otherwise TS/ESLint will complain if an explicit property isn't passed, I fixed the other inputs in https://github.com/viamrobotics/prime/pull/326

Reply:
This is just for a test element but sure.

Reply:
Tried it on both and am getting this linting error:

`'any' overrides all other types in this union type  @typescript-eslint/no-redundant-type-constituents`

Reply:
that's... odd, there aren't any `any`s in that union?